src/main/java/com/palantir/docker/proxy/DockerProxyRule.java

```diff
@@ -196,7 +196,7 @@ private static Object wrapNameService(String className, Object delegate, Object
         }
 
         @Override
-        public Object invoke(Object proxy, Method method, Object[] args) throws Throwable {
+        public Object invoke(Object _proxy, Method method, Object[] args) throws Throwable {
```

Comment:
Does checkstyle accept this being `proxy`?

Reply:
no, this is new